Dockerfile

```
@@ -41,6 +41,10 @@ RUN curl -F inputShapeFile=@profil-application/rdafr-shacl.ttl \
          -H 'Accept-Language: fr-FR,fr' \
          https://shacl-play.sparna.fr/play/doc \
          > /usr/share/nginx/html/profil-application/index.html
+
+# Suppression des URLs non déréférençables
+RUN sed -i -E 's/<a href="(https:\/\/rdafr\.fr\/Elements\/.*?\/)" target="_blank">.*?<\/a>/\1/;s/<a href="https:\/\/rdafr\.fr\/(Elements|termList)\/.*?>(.*?)<\/a>/\2/' /usr/share/nginx/html/profil-application/index.html
```

En découpant le sed sur deux lignes, ca donnerait ceci ? 
```suggestion
RUN sed -i -E 's#<a href="(https://rdafr\.fr/Elements/.*?/)" target="_blank">.*?</a>#\1#' /usr/share/nginx/html/profil-application/index.html
RUN sed -i -E 's#<a href="https://rdafr\.fr/(Elements|termList)/.*?>(.*?)</a>#\1#' /usr/share/nginx/html/profil-application/index.html
```

---

J'avais complètement oublié la possibilité d'utiliser d'autres délimiteurs pour les expressions sed. L'utilisation de `#` rend les choses beaucoup plus lisible 👍.

Pour la scission j'ai opté pour une solution alternative qui nous évite d'avoir à lire le fichier deux fois (même si en terme de performance c'est négligeable).

```
RUN sed -E -i /usr/share/nginx/html/profil-application/index.html \
    -e 's#<a href="(https://rdafr\.fr/Elements/.*?/)" target="_blank">.*?</a>#\1#' \
    -e 's#<a href="https://rdafr\.fr/(Elements|termList)/.*?>(.*?)</a>#\2#'
```

---

Ha excellent le double "-e", je ne connaissais pas.